R/tbl_summary.R

```diff
@@ -175,8 +175,9 @@ tbl_summary <- function(data, by = NULL, label = NULL, statistic = NULL,
   value <- value %||% get_theme_element("tbl_summary-arg:value")
   missing <- missing %||% get_theme_element("tbl_summary-arg:missing",
                                             default = "ifany")
-  missing_text <- missing_text %||% get_theme_element("tbl_summary-arg:missing_text",
-                                                      default = "Unknown")
+  missing_text <- missing_text %||%
+    get_theme_element("tbl_summary-arg:missing_text",
```

Comment:
@oranwutang i added translation for the "Unknown" in `tbl_summary()` output.  Can you add the Spanish translation?

Reply:
Done!